chains/evm/voter/voter.go

```diff
@@ -159,6 +165,19 @@ func (v *EVMVoter) shouldVoteForProposal(prop *proposal.Proposal, tries int) (bo
 	return true, nil
 }
 
+func (v *EVMVoter) simulateVoteProposal(prop *proposal.Proposal, tries int) error {
```

Comment:
Please add godoc comments, as well as i think that function name should be more self expressed, like 
repetitiveSimulateVote because it i obviously  not just a simulate call, but also  includes additional logic